app/api/client_resources.py

```diff
@@ -283,7 +283,7 @@ def resource_update(
                 "id": resource.id,
                 "name": resource.name,
                 "address": resource.address,
-                "alias": resource.alias,
+                "alias": "" if resource.alias is None else resource.alias,
```

Comment:
`None` does nothing to the `alias`, we need to set it to empty string to remove the `alias`